Message.php

```diff
@@ -233,7 +238,26 @@ protected function setBody($body, $contentType)
             }
         }
     }
-
+    
+    /**
+     * Adds a Swift_Signers_DKIMSigners object to the Swift_SignedMessage message.
+     * This requires the messageClass to be set to \Swift_SignedMessage instead of \Swift_Message.
+     * messageClass can be set to the correct value via yii\swiftmailer\Mailer::$messageConfig
+     * @param string $privateKeyAlias an alias pointing to the private key file.
+     * @param string $domain the domain to sign with.
+     * @param string $selector the dkim domain selector.
+     */
+     public function setDkim($privateKeyAlias, $domain, $selector) {
+        $privateKey = file_get_contents(Yii::getAlias($privateKeyAlias));
```

Comment:
$privateKey = file_get_contents(\Yii::getAlias($privateKeyAlias)); //slash was missing in front of Yii as you did not defined "use yii;"